api/src/main/java/org/openmrs/module/pihcore/setup/LiquibaseSetup.java

```diff
@@ -6,16 +6,26 @@
 import org.openmrs.api.context.Context;
 import org.openmrs.liquibase.ChangeSetExecutorCallback;
 import org.openmrs.module.pihcore.PihCoreUtil;
+import org.openmrs.module.pihcore.config.Config;
+import org.openmrs.module.pihcore.config.model.PihConfig;
 import org.openmrs.util.DatabaseUpdater;
+import org.openmrs.util.OpenmrsUtil;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
 
 public class LiquibaseSetup {
 
     protected static Log log = LogFactory.getLog(LiquibaseSetup.class);
 
-    public static void setup() throws Exception {
+    public static void setup(Config config) throws Exception {
         try {
             updateLiquibaseChangeLogPath();
             DatabaseUpdater.executeChangelog(PihCoreUtil.getLiquibaseChangeLog(), (ChangeSetExecutorCallback) null);
+            if (new File(OpenmrsUtil.getApplicationDataDirectory()  + "/" + PihCoreUtil.getSiteSpecificChangeLog(config)).exists()) {
```

Comment:
I would think this should be new "File(OpenmrsUtil.getApplicationDataDirectory(), PihCoreUtil.getSiteSpecificChangeLog(config))" - eg. just separate the dir and file with a comma as separate arguments.